presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestGrantRevoke.java

```diff
@@ -68,6 +71,14 @@ public void testGrantRevoke()
         assertThat(() -> bobExecutor.executeQuery(format("DELETE FROM %s WHERE day=3", tableName))).
                 failsWithMessage(format("Access Denied: Cannot delete from table default.%s", tableName));
 
+        // test SHOW GRANTS
```

Comment:
Please make a new test -- it doesn't cost us much, and then if there's a failure, it'll be clear what is the problem.